README.md

```diff
@@ -1,61 +1,46 @@
-# Azure Functions Serverless Plugin 
+# Azure Functions Serverless Plugin
```

Comment:
We'll need to add more in addition to the quickstart.  But this is good for now.